src/mom5/ocean_core/ocean_sbc.F90

```diff
@@ -2941,6 +2979,12 @@ subroutine sum_ocean_sfc(Time, Thickness, T_prog, T_diag, Dens, Velocity, Ocean_
 #if defined(ACCESS_CM) || defined(ACCESS_OM)
             Ocean_sfc%gradient(i,j,:) = Ocean_sfc%gradient(i,j,:) + sslope(ii,jj,:)  
 #endif
+#if defined(ACCES_OM) && defined(CSIRO_BGC)
```

Comment:
Thanks for spotting this bug! I was actually looking for this for a while, because my recent runs had been getting zero values in sea-ice BGC tracers. As you see below, I have pushed the bug fix.